### PR TITLE
check record creation time too to mark it stalled

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/77988

Work in progress for the ticket.

In this PR:
1. allow an env var to override schedule checks; useful to force scheduled imports to rerun using wp-cli like `export TRIBE_DEBUG_OVERRIDE_SCHEDULE=1 && wp cron event run tribe_aggregator_cron --debug`
2. adds a check on the creation time too for the records to mark records as stalled if not complete after 12 hrs

Building on https://github.com/moderntribe/tribe-common/pull/382 (Meta Chunker) now the items are stored and retrieved with the size in mind.

### CODE REVIEW

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 1]
```diff
@@ -792,6 +792,10 @@ public function log_error( $error ) {
 	 * @return boolean
 	 */
 	public function is_schedule_time() {
+		if ( true == getenv( 'TRIBE_DEBUG_OVERRIDE_SCHEDULE' ) ) {
```
Comment (bordoni) [inline src/Tribe/Aggregator/Record/Abstract.php]: We should use `tribe_is_truthy` here.

**Comment by lucatume** [comment]
I've added a second check in the cron process too to purge stalled records on cron runs too; was only on ajax before.

**Comment by lucatume** [comment]
Putting this on Hold as QA is happening in branch

**Comment by lucatume** [comment]
After using the code provided in https://github.com/moderntribe/tribe-common/pull/382 this is ready for another pass.

**Comment by lucatume** [comment]
Andras has QAed this, good for review.

**Comment by lucatume** [comment]
Closing this to move PR to MR here https://github.com/moderntribe/the-events-calendar/pull/1383

**Pull request closed without merging**